### PR TITLE
Reducers and actions PR

### PR DESCRIPTION
**Pull request by kkasaija**
- Added redux tool kit
- Wrote  book's actions and reducer
- Wrote categories actions and reducer
- Configured redux store

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const ADD_BOOK = 'ADD_BOOK';
+const DELETE_BOOK = 'DELETE_BOOK';
```
Comment (emmiiorji) [inline src/redux/books/books.js]: Your code structure looks great. Kindly ensure that each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Example:
`const ADD_BOOK = bookstore/books/ADD_BOOK`
`const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

Following this pattern will help you know at a glance which reducer has the action.

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const CATEGORIES_STATUS = 'CATEGORIES_STATUS';
```
Comment (emmiiorji) [inline src/redux/categories/categories.js]: As suggested above and for the same reasons, kindly utilize the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for defining your action types.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Kasaija-Kenneth, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

a little more

## Good Highlights✔

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 2 above)
Comment (kkasaija) [reply src/redux/categories/categories.js]: Noted with thanks

File: src/redux/books/books.js (hunk 1 above)
Comment (kkasaija) [reply src/redux/books/books.js]: Noted with thanks

**Review by brytebee — APPROVED** [review]
Hi, @Kasaija-Kenneth👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



##

###  **Optional Suggestions:**
- N/A

##

Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkasaija** [comment]
Much appreciated. 🙏🙏🙏🙏

**Pull request merged**